inc/field/dropdownfield.class.php

```diff
@@ -266,7 +267,7 @@ public function buildParams($rand = null) {
                   ];
                }
                // Check if helpdesk availability is fine tunable on a per item basis
-               if (Session::getCurrentInterface() == "helpdesk" && $DB->fieldExists($itemtype::getTable(), 'is_helpdesk_visible')) {
+               if ($DB->fieldExists($itemtype::getTable(), 'is_helpdesk_visible') && count($target_ticket) > 0) {
```

Comment:
is the count() necessary? wouldn't that be enough?:
```suggestion
               if ($DB->fieldExists($itemtype::getTable(), 'is_helpdesk_visible')) {
```

Reply:
This allows you to check whether the form is intended to create a ticket or not. If a form is not intended to create a ticket, then I take into account the elements that are not associable to a ticket.